src/AspNet.Security.OAuth.Kook/AspNet.Security.OAuth.Kook.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
+
+  <!-- TODO Remove once published to NuGet.org -->
+  <PropertyGroup>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+  </PropertyGroup>
```

Comment:
The baseline needs to be 7.0.1, then after publishing we'd only remove `DisablePackageBaselineValidation` and not the baseline property.

```suggestion
  <PropertyGroup>
    <!-- TODO Remove once published to NuGet.org -->
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
    <PackageValidationBaselineVersion>7.0.1</PackageValidationBaselineVersion>
  </PropertyGroup>
```